test/testgarbage.cpp

```diff
@@ -1451,6 +1452,13 @@ class TestGarbage : public TestFixture {
                   "}");
     }
 
+    void garbageCode191() { // #8333
+        ASSERT_THROW(checkCode("struct A { int f(const); };"), InternalError);
+        ASSERT_THROW(checkCode("struct A { int f(int, const, char); };"), InternalError);
+        ASSERT_THROW(checkCode("struct A { int f(struct); };"), InternalError);
+        checkCode("struct A { int f(int); };");
```

Comment:
this code is fine as far as I know so I guess there should be a comment that says so. As far as I know all other code examples in this file are garbage.

Reply:
It's invalid code except for the fourth test case, but I agree it's not exactly garbage. Moved to tokenizer tests